.github/workflows/pr-tests.yml

```
@@ -49,6 +50,36 @@ jobs:
+        continue-on-error: true
+
+      - name: Run e2e tests
+        id: e2e
+        run: xvfb-run --auto-servernum npm run test:e2e
+        env:
+          CI: true
+        continue-on-error: true
+
+      - name: Upload e2e test results
+        if: always() && steps.e2e.outcome != 'skipped'
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.10)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)